### DISCUSSION TITLE
rAudio 1 20220513 mpd daemon doesn't start after update (mpd.log permission denied)

### DISCUSSION THREAD

**Discussion by somax80** [rern/rAudio#926] [Bug Reports]
Hi,
after update 20220513 release (and also after update of 20220428 release), mpd daemon doesn't start up.
The error message is:
`runeaudio taskset[1520]: exception: failed to open log file "/srv/http/data/mpd/mpd.log" (config line 11): Permission denied `

mpd.log file:
`-rwxr-xr-x 1 http http 189840608 Apr 30 09:16 mpd.log `

If I manually chown mpd.log to mpd:audio, then daemon starts correctly.

I've seen that mpd service runs as user "mpd" (mpd group), that doesn't have permissions to write on mpd.log.
`mpd 663 0.5 3.9 205608 38588 ? S<sl 09:23 0:01 /usr/bin/mpd --no-daemon `
The owner of mpd.log file was changed automatically in "http" by the installation script.

Many thanks
Max

**Comment by ceejayemm** [comment]
After applying update 20220513 I am not seeing this issue, the mpd daemon starts up as expected. I do not have an mpd.log file anywhere on my system.  Have you customised the mpd setup in some way that is trying to create this file ?

Chris

**Reply by somax80** [reply]
> After applying update 20220513 I am not seeing this issue, the mpd daemon starts up as expected. I do not have an mpd.log file anywhere on my system. Have you customised the mpd setup in some way that is trying to create this file ?
> 
> Chris

You're right!

A long time ago when I imported my music library, I had to turn on file log and verbose mode to investigate an ffmpeg problem.

I had set the log file path under /srv/http/data/mpd/, so that was the problem.
Now I've moved it under /tmp/ and the service starts up fine.

Thanks for your suggestion
Max

**Comment by cmh714** [comment]
I have this issue as well, just never reported it....reboot fixes it, but yeah its a little annoying.
